docs/docs/topics/infrahub-yml.mdx

```diff
@@ -60,6 +60,33 @@ Schemas to be loaded as part of an external repository can be defined in file(s)
 
 Infrahub will attempt to import any schemas defined in `.infrahub.yml` when pulling from the external repository.
 
+## Objects
```

Comment:
I think it's important that we mention something about ordering here.

AFAIK the order in which we define the objects files determines the order they are loaded, which is important because some object files could rely on the fact that some objects were already loaded.

Did we test this?

Reply:
It is indeed the user-defined order that is used.